### PR TITLE
Add preferred roulette command option

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

### CODE REVIEW

**Review by RAnders00 — CHANGES_REQUESTED** [review]
Could you just make it a text setting? E.g. like here: https://github.com/pajbot/pajbot/blob/d0304c7433fead71f64438d7ff5d2da81bca0043/pajbot/modules/paidtimeout.py#L23-L31

**Review by RAnders00 — CHANGES_REQUESTED** [review]
You also need to make the examples dynamic, !roulette is currently hardcoded

**Review by RAnders00 — APPROVED** [review]

**Review by pajlada — APPROVED** [review]

File: pajbot/modules/roulette.py [hunk 1]
```diff
@@ -24,6 +24,15 @@ class RouletteModule(BaseModule):
     DESCRIPTION = "Lets players roulette with themselves for points"
     CATEGORY = "Game"
     SETTINGS = [
+        ModuleSetting(
+            key="command_name",
+            label="Command name (i.e. roulette)",
```
Comment (pajlada) [inline pajbot/modules/roulette.py:29]:
```suggestion
            label="Command name (e.g. roulette)",
```

Comment (pajlada) [reply pajbot/modules/roulette.py:29]: minor thing but pls accept this @alazymeme and I'll merge it

**Review by pajlada — APPROVED** [review]

**Pull request merged**